.azurepipelines/Platform-Build-Job.yml

```diff
@@ -19,120 +19,172 @@ parameters:
+        and(not(Canceled()),
+            or(${{ ne(item.Value.SelfHostAgent, true) }}, ne(variables['${{ parameters.pool_name }}'], '')))
+
+      workspace:
+        clean: all
+
+      variables:
+      - name: Run
+        ${{ if or(eq(item.Value.Run, true), eq(parameters.os_type, 'Linux')) }}:
+          value: true
+        ${{ else }}:
+          value: false
+
+      ${{ if and(ne(parameters.container_image, ''), ne(item.Value.SelfHostAgent, true)) }}:
+        container: ${{ parameters.container_image }}
+
+      ${{ if eq(item.Value.SelfHostAgent, true) }}:
+        pool:
+          name: $(${{ parameters.pool_name }})
+          demands:
+          - Agent.OS -equals ${{ parameters.os_type }}
+      ${{ else }}:
+        pool:
+          vmImage: ${{ parameters.vm_image }}
+
+      steps:
+      - ${{ if and(and(ne(parameters.container_image, ''), ne(item.Value.SelfHostAgent, true)), not(contains(parameters.vm_image, 'windows'))) }}:
+        - script: echo "##vso[task.prependpath]/home/vsts_azpcontainer/.local/bin"
+          displayName: Add User Local Bin to Path
+        - script: sudo dnf install -y mtools dosfstools
+          displayName: Install mtools and dosfstools
+
+      # Potential Extra steps from specific build targets
+      - ${{ item.Value.BuildExtraStep }}
+
+      - template: Steps/BuildPlatform.yml@mu_devops
+        parameters:
+          tool_chain_tag: $(tool_chain_tag)
+          build_pkg: ${{ item.Value.BuildPackage }}
+          build_target: ${{ item.Value.BuildTarget }}
+          build_arch: ${{ item.Value.BuildArch }}
+          build_file: ${{ item.Value.BuildFile }}
+          build_flags: ${{ item.Value.BuildFlags }}
+          run_flags: ${{ item.Value.RunFlags }}
+          run_timeout: ${{ parameters.run_timeout }}
+          extra_install_step: ${{ parameters.extra_install_step }}
+          install_tools: ${{ and(ne(item.Value.SelfHostAgent, true), eq(parameters.container_image, '')) }}
+          artifacts_identifier: '${{ item.Value.BuildPackage }} $(tool_chain_tag) ${{ item.Value.BuildTarget }} ${{ item.Value.BuildExtraTag }}'
+          artifacts_binary: ${{ item.Value.BuildArtifactsBinary }}
+          ${{ if or(eq(item.Value.Run, true), eq(parameters.os_type, 'Linux')) }}:
+            artifacts_other: ${{ item.Value.BuildArtifactsOther }}
+
+    - job: RunQemuBuilds_${{ item.Key }}
+      displayName: Run Qemu Builds ${{ item.Key }}
+      dependsOn: PlatformBuild_${{ item.Key }}
+      condition: |
+        and(succeeded(),
+            and(ne(variables['tool_chain_tag'], 'GCC5'),
+                and(StartsWith('${{ item.Key }}', 'QemuQ35'), EndsWith('${{ item.Key }}', 'ARM'))))
+
+      pool:
+        vmImage: ${{ parameters.vm_image }}
+
+      steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 1
+
+      - template: Steps/SetupPythonPreReqs.yml@mu_devops
+        parameters:
+          install_python: true
+
+      # Set default
+      - powershell:

```

Comment:
Could this be just a normal script and echo so we dont further our dependency on powershell?

Reply:
Yeah, I will try to change them to `script echo`

Reply:
Updated. Seems to be doing fine. Thanks!